app-utils/chafa/autobuild/postinst

```diff
@@ -0,0 +1 @@
+ldconfig
```

Comment:
Why?

Reply:
I wasn't sure if ab4 could do this automatically, so I added this script.

![image](https://github.com/user-attachments/assets/55c8d754-6712-4ef3-9cd4-e97933eaed74)

Reply:
> I wasn't sure if ab4 could do this automatically, so I added this script.
> 
> ![image](https://private-user-images.githubusercontent.com/91523109/353530699-55c8d754-6712-4ef3-9cd4-e97933eaed74.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIzOTY3NjEsIm5iZiI6MTcyMjM5NjQ2MSwicGF0aCI6Ii85MTUyMzEwOS8zNTM1MzA2OTktNTVjOGQ3NTQtNjcxMi00ZWYzLTljZDQtZTk3OTMzZWFlZDc0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDAzMjc0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2YjhmYThmODI4MzFkMThjM2RlMjM0ODgzODU3MzBiNDY5NGVhYWI5Zjg1YjFhMjUxYzQxNTMxMmM0NzNiMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.drtKwtx8cg5c_d0hbBnSftIsREyMpSJXqr0dVFwJ-3I)

Definitely not necessary on AOSC OS unless the libraries (if any) were installed in non-standard locations.

Reply:
Removed, thanks!